conductr_cli/conduct_main.py

```diff
@@ -494,6 +495,19 @@ def build_parser(dcos_mode):
 
     add_default_arguments(backup_parser, dcos_mode)
     backup_parser.set_defaults(func=conductr_backup.backup)
+
+    restore_parser = subparsers.add_parser('restore',
+                                           help='Restore a cluster from backup.',
+                                           formatter_class=argparse.RawTextHelpFormatter)
+
+    restore_parser.add_argument('backup',
+                                default=None if sys.stdin.isatty() else '-',
```

Comment:
I think if you add `nargs=None if sys.stdin.isatty() else '?',` we will be all set -- this should ensure `conduct backup | conduct load` works.